redis/src/main/scala/zio/redis/RedisEnvironment.scala

```diff
@@ -0,0 +1,8 @@
+package zio.redis
+
+import zio.schema.codec.BinaryCodec
+
+trait RedisEnvironment {
```

Comment:
Maybe we should make it package-private. After all, it is an implementation detail.

Reply:
yes, makes sense